migrations/2021-08-23-161500_aggregated_ft_nft_tables/up.sql

```diff
@@ -0,0 +1,62 @@
+CREATE TABLE aggregated__fungible_tokens
+(
+    id                                bigserial                NOT NULL,
+    included_in_transaction_hash      text                     NOT NULL,
+    included_in_transaction_timestamp numeric(20, 0)           NOT NULL,
+    transaction_status                execution_outcome_status NOT NULL,
+    issued_contract_id                text                     NOT NULL,
+    called_method                     text                     NOT NULL,
+    predecessor_account_id            text                     NOT NULL,
+    receiver_account_id               text                     NOT NULL,
```

Comment:
From the "fungible token" table point of view these are ambiguous names since `receiver_account_id` of a transaction, a receipt, and FT token are three different account ids, and I feel that in the context of "fungible token" table we care about the FT sender and FT receiver account ids and the rest is less relevant.